src/index.d.ts

```diff
@@ -5,8 +5,8 @@ export interface VerificationInputProps {
   placeholder?: string;
   autoFocus?: boolean;
   passwordMode?: boolean;
-  inputProps?: object;
-  containerProps?: object;
+  inputProps?: React.HTMLAttributes<HTMLInputElement>;
+  containerProps?: React.HTMLAttributes<HTMLDivElement>;
```

Comment:
Thanks, very nice.
However, the attributes don't seem to be complete. For example, `disabled` is missing for the input element.
I found `React.ComponentPropsWithRef<"input">` which is supposed to be exactly the props you can pass to an `<input />`. What do you think?

Reply:
Since we just pass along everything this also includes the ref. Forward ref is only required for custom components, all the built-in components like input and div support it out of the box.